README.md

```diff
@@ -1 +1,102 @@
+CHT Upgrade Service
+========================
 
+The CHT Upgrade Service provides an interface between CHT-Core API and Docker to allow easy startup and one-click upgrades from the CHT-Core Admin UI. 
+
+# Requirements
+
+* Docker
+* docker-compose
+
+# Installation and usage
+
+Installation and usage is achieved entirely with docker-compose. 
+
+#### Download and save CHT-Core docker-compose yml file. 
+
+Todo steps??
+
+#### Download and save CHT upgrade service docker-compose yml file.
+
+Todo steps??
+
+#### Export the environment variables:
+
+| Name                 | Required | Description                                                                                                        |
+|----------------------|----------|--------------------------------------------------------------------------------------------------------------------|
+| `DOCKER_CONFIG_PATH` | no       | Absolute path to your docker-config file to allow access to authenticated AWS ECR endpoints to pull private images | 
```

Comment:
Move this one down so the required variables are first. Also document what happens if you don't specify it, presumably you can only get public images?